canvas_modules/common-canvas/src/common-properties/controls/checkboxset/checkboxset.jsx

```diff
@@ -21,9 +21,13 @@ import { Checkbox } from "carbon-components-react";
 import * as ControlUtils from "./../../util/control-utils";
 import classNames from "classnames";
 import ValidationMessage from "./../../components/validation-message";
-import { STATES } from "./../../constants/constants.js";
 import { v4 as uuid4 } from "uuid";
 import { intersection, isEqual } from "lodash";
+import Icon from "./../../../icons/icon.jsx";
```

Comment:
You should be using `@carbon/icons-react` instead of icon.jsx.  We want to eventually remove icon.jsx since it doesn't work well with tree shaking.

Reply:
Used **Information16** instead 
<img width="458" alt="image" src="https://user-images.githubusercontent.com/89609176/174674810-1a10681f-1496-452d-9cc8-c6ba04d70c8a.png">